### ISSUE TITLE
Touchscreen and Overlays - Touchscreen doesn't work (supported??)

### ISSUE THREAD

**Issue by andymcca** [mac2612/retroleap#24]
Hiya

Before I start fiddling :-), are the LeapPad touch screens supported and, by extension, overlays to allow extra 'keys' to be displayed and utilised from the touch screen?  If the answer to both is no then I'll start having a dig around to see what I can do.

The Leapster Explorer and Leapster GS are great in that they have a similar button layout to a GBA, but it would be nice to have more buttons on the LeapPad 3 especially, given the superior hardware.

Thanks
Andy

**Comment by CE1CECL** [comment]
> Hiya
> 
> Before I start fiddling :-), are the LeapPad touch screens supported and, by extension, overlays to allow extra 'keys' to be displayed and utilised from the touch screen? If the answer to both is no then I'll start having a dig around to see what I can do.
> 
> The Leapster Explorer and Leapster GS are great in that they have a similar button layout to a GBA, but it would be nice to have more buttons on the LeapPad 3 especially, given the superior hardware.
> 
> Thanks Andy

@andymcca yeah, idk if its supported b/c https://github.com/mac2612/retroleap/blob/811626c957a8b7d60eab1e5bcb68c0ded9a57b66/README says "-Overlay support for the tablet form-factor devices to make up for lack of buttons." and was there since  Dec 9, 2018, which most likely means the same thing as this issue. Does it need non-free firmware?
